recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -461,8 +461,10 @@ install:
               echo "{\"Metadata\":{\"previousNriBundleDetected\":\"No\"}}" | tee -a {{.NR_CLI_OUTPUT}} > /dev/null
             fi
 
-            if [[ $($SUDO helm repo add newrelic https://helm-charts.newrelic.com) ]]; then
-            else
+            HELM_VERSION="$($SUDO helm version -c --short)"
+            ERROR=$($SUDO helm repo add newrelic https://helm-charts.newrelic.com)
+            if [[ "${ERROR}" != "" && "${ERROR}" != "\"newrelic\" already exists with the same configuration, skipping" ]]; then
```

Comment:
i saw your screenshotted error message, but do you have an extra " in the "\"newrelic\" here?

Reply:
oh nevermind, i see what you're doing, saw the order wrong